src/StaticAnalysisResultsInterchangeFormat/GrammarFiles/SarifFormat.g4

```diff
@@ -124,8 +127,8 @@ runInfo :
         @name {AnalysisTargets}
         @summary {
 		An array, each of whose elements is a fileReference object (�6.7) representing the location of 
```

Comment:
Should we include these paragraph references? They're not clickable here; how valuable are they?